### DISCUSSION TITLE
Properly installing `wfxr/forgit`

### DISCUSSION THREAD

**Discussion by norman-zon** [marlonrichert/zsh-snap#275] [Q&A] [answered]
I have issues installing [forgit](https://github.com/wfxr/forgit).
When I do 

```zsh
znap source wfxr/forgit
```

neither `/completions` are added to `fpath` nor `/bin` to my path.

What am I missing? Isn't this what `znap source` is supposed to do?

**Comment by marlonrichert** [comment, marked as answer]
Znap intentionally doesn't do this. Either the plugin itself should do this (if you think so, then open an issue for it at `wfxr/forgit`) or you can add this to your dotfiles:
```sh
znap source wfxr/forgit
path+=(  ~[wfxr/forgit]/bin )
fpath+=( ~[wfxr/forgit]/completions )
```
